src/db.c

```diff
@@ -228,6 +228,7 @@ static const struct col_type_map mfi_cols_map[] =
     { "composer_sort",      mfi_offsetof(composer_sort),      DB_TYPE_STRING, DB_FIXUP_COMPOSER_SORT },
     { "channels",           mfi_offsetof(channels),           DB_TYPE_INT },
     { "usermark",           mfi_offsetof(usermark),           DB_TYPE_INT },
+    { "source",             mfi_offsetof(source),             DB_TYPE_STRING, DB_FIXUP_NO_SANITIZE },
```

Comment:
Is the name "source" a bit too broad? At first I was thinking "source of what?". What do you think about library_source instead? Could also make it easier to grep for when one wants to see where it is used.

Reply:
I agree, "source" is probably not specific enough. I changed it like you suggested to "library_source". We have a minor naming clash now, because there is already a struct with the name `library_source`. But I failed to come up with a better naming. If you have better ideas let me know and I change it.

Reply:
Yes, didn't consider the struct. Now that you mention data_kind/media_kind then an alternative could be library_kind? Or scan_kind?